code/modules/mob/living/living.dm

```diff
@@ -1675,6 +1675,13 @@
 			if(old_buckled.buckle_lying == 0 && (resting || HAS_TRAIT(src, TRAIT_FLOORED))) // The buckle forced us to stay up (like a chair)
 				set_lying_down() // We want to rest or are otherwise floored, so let's drop on the ground.
 
+///can the mob be buckled to something by default?
+/mob/proc/can_buckle()
+	return TRUE
+
+///can the mob be unbuckled from something by default?
+/mob/proc/can_unbuckle()
+	return TRUE
```

Comment:
Could you move these procs into variables, or even better, a single bitflag variable? Something like `buckling_flags`?
Check the contribution guidelines on complex and redefined getters, for an example on how to better handle this kind of thing.

Reply:
Sure

Reply:
Oh man I didn't even notice but I forgot to change it to mob/living too lol